VERSION

```diff
@@ -1,2 +1,2 @@
 MAJOR=0
-MINOR=143
+MINOR=154
```

Comment:
This part is wrong. This value is incremented automatically whenever we make a new canary release, so it should not be touched by hand.